### PR TITLE
Send test emails only to email put in form

### LINKED ISSUE #203: Allow test emails to be sent to the email put in the form
When conducting tests (proca_test get param), the mail2target actions don't send the emails to the real targets but to a test fixed test email. It makes it hard to validate the emails for the campaigner and creates confusion

The ideal scenario is that the campaigner puts their own email address as the supporter (jane@example.org) and she receives the thank you email (if set up) AND the email that was supposed to be sent to the target

to make it clearer that the second email is a test and isn't meant to be the one they receive normally, we should add a "[TEST]" at the start of the subject line.

the test emails aren't following the "normal" drip delivery configurations and are sent as soon as possible

### PR DESCRIPTION
**Pull request by destag**
Fix #203

### CODE REVIEW

**Comment by destag** [comment]
Opened to start a discussion, it should not be merged in this form

File: proca/lib/proca/server/mtt_worker.ex [hunk 1]
```diff
@@ -315,6 +315,7 @@ defmodule Proca.Server.MTTWorker do
       {:mtt, message_id}, email_to.id
     )
     |> Email.from({supporter_name, supporter.email})
+    |> Email.cc(test_email)
```
Comment (destag) [inline proca/lib/proca/server/mtt_worker.ex:318]:
```elixir
iex> Swoosh.Email.new(to: {"john doe", "john.doe@example.com"}) |> Swoosh.Email.cc(nil) |> Map.take([:to, :cc])
%{cc: [], to: [{"john doe", "john.doe@example.com"}]}
iex> Swoosh.Email.new(to: {"john doe", "john.doe@example.com"}) |> Swoosh.Email.cc("test@example.com") |> Map.take([:to, :cc])
%{cc: [{"", "test@example.com"}], to: [{"john doe", "john.doe@example.com"}]}
```

Comment (tttp) [reply proca/lib/proca/server/mtt_worker.ex:318]: @destag, it should only be there when running in test mode, not for real one, of course

Comment (destag) [reply proca/lib/proca/server/mtt_worker.ex:318]: You're right, I overlooked that. We can have test_email set but not be in test mode

File: proca/lib/proca_web/resolvers/action.ex [hunk 2]
```diff
@@ -172,7 +172,20 @@ defmodule ProcaWeb.Resolvers.Action do
            get_action_page(params)
          end)
          |> Multi.run(:supporter, fn _repo, %{action_page: action_page} ->
-           get_supporter(action_page, params)
+           case get_supporter(action_page, params) do
+             {:ok, supporter = %Supporter{}} ->
+               {:ok, supporter}
+
+             {:ok, cref} ->
+               if Map.has_key?(params.action, :mtt) do
```
Comment (destag) [inline proca/lib/proca_web/resolvers/action.ex:180]: I think we can also just check if action_type == "mtt"
@tttp What do you think?

**Pull request merged**